docs/index.md

```diff
@@ -1,5 +1,17 @@
 # Flux documentation
 
+!!!info
```

Comment:
You may want to give this block a title using `!!!info "<title>"`

Reply:
nice one

Reply:
(You may also want to change it into a `!!!warning`)

Reply:
(or whatever the warning block title was again)